server/build.gradle.kts

```diff
@@ -131,29 +116,17 @@ tasks {
     shadowJar {
         manifest {
             attributes(
-                mapOf(
-                    "Main-Class" to MainClass,
-                    "Implementation-Title" to rootProject.name,
-                    "Implementation-Vendor" to "The Suwayomi Project",
-                    "Specification-Version" to tachideskVersion,
-                    "Implementation-Version" to tachideskRevision
-                )
+                "Main-Class" to MainClass,
+                "Implementation-Title" to rootProject.name,
+                "Implementation-Vendor" to "The Suwayomi Project",
+                "Specification-Version" to tachideskVersion,
+                "Implementation-Version" to tachideskRevision
             )
         }
         archiveBaseName.set(rootProject.name)
         archiveVersion.set(tachideskVersion)
         archiveClassifier.set(tachideskRevision)
     }
-    withType<KotlinCompile> {
-        kotlinOptions {
-            freeCompilerArgs = listOf(
-                "-Xopt-in=kotlin.RequiresOptIn",
-                "-Xopt-in=kotlinx.coroutines.ExperimentalCoroutinesApi",
-                "-Xopt-in=kotlinx.coroutines.InternalCoroutinesApi",
-                "-Xopt-in=kotlinx.serialization.ExperimentalSerializationApi",
-            )
-        }
-    }
```

Comment:
we don't need this anymore?

Reply:
It was moved to the top-level build.gradle.kts